python/cudf/cudf/core/series.py

```diff
@@ -6019,6 +6019,44 @@ def explode(self, ignore_index=False):
 
         return super()._explode(self._column_names[0], ignore_index)
 
+    def pct_change(
+        self, periods=1, fill_method="ffill", limit=None, freq=None
+    ):
+        """
+        Calculates the percent change between sequential elements
+        in the Series.
+
+        Parameters
+        ----------
+        periods : int, default 1
+            Periods to shift for forming percent change.
+        fill_method : str, default 'ffill'
+            How to handle NAs before computing percent changes.
+        limit : int, optional
+            The number of consecutive NAs to fill before stopping.
+            Not yet implemented.
+        freq : str, optional
+            Increment to use from time series API.
+            Not yet implemented.
+
+        Returns
+        -------
+        Series
+        """
+        if limit is not None:
+            raise NotImplementedError("limit parameter not supported yet.")
+        if freq is not None:
+            raise NotImplementedError("freq parameter not supported yet.")
+        if fill_method not in ["ffill", "bfill"]:
+            raise NotImplementedError(
```

Comment:
Argument out of specified range is `ValueError`
```suggestion
            raise ValueError(
```